Jenkinsfile

```diff
@@ -261,7 +261,7 @@ spec:
                         container('jnlp') {
                             script {
                                 uploadInstaller('windows')
-                                copyInstallerAndUpdateLatestYml('windows', 'CDTCloudBlueprintSetup', 'exe', 'latest.yml', '1.40.1')
+                                copyInstallerAndUpdateLatestYml('windows', 'CDTCloudBlueprintSetup', 'exe', 'latest.yml', '1.46.0,1.46.100,1.47.0,1.47.100,1.48.0,1.48.300,1.49.100')
```

Comment:
I think we should only list versions here which we actually released, see [here](https://download.eclipse.org/theia/cdt-cloud/)

Reply:
Thanks for the link, I've updated the list of released versions accordingly.